src/parse_styles.rs

```diff
@@ -28,8 +28,14 @@ pub fn parse_styles(opt: &cli::Opt) -> HashMap<String, Style> {
     make_misc_styles(opt, &mut styles);
 
     let mut resolved_styles = resolve_style_references(styles, opt);
-    resolved_styles.get_mut("minus-emph-style").unwrap().is_emph = true;
-    resolved_styles.get_mut("plus-emph-style").unwrap().is_emph = true;
+    resolved_styles
+        .get_mut("minus-emph-style")
+        .expect("minus-emph-style not found in resolved styles")
+        .is_emph = true;
+    resolved_styles
+        .get_mut("plus-emph-style")
+        .expect("plus-emph-style not found in resolved styles")
+        .is_emph = true;
```

Comment:
The project has a (admittedly idiosyncratic) style of using `unwrap_or_else(|| panic(...))` instead of `expect(..)` because of the confusion regarding how to read and write `expect` messages (e.g. what you wrote vs "minus-emph-style should be in resolved styles"). Would you mind switching to use `unwrap_or_else`?

Reply:
hello! sure, corrected